### PR TITLE
TEST Data build fix-ncit-synonyms feature branch

### PR DESCRIPTION
**Pull request by twhetzel**
Resolves #ISSUE(s).

Related to:
- Issue: https://github.com/monarch-initiative/mondo-ingest/issues/612
- PR: https://github.com/monarch-initiative/mondo-ingest/pull/613

## Overview

This PR:
Test feature branch build for `fix-ncit-synonyms` branch.

### Documentation
Was the documentation added/updated under `docs/`?
- [ ] Yes
- [x] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [x] Yes
- [ ] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [ ] Yes

### CODE REVIEW

File: docs/reports/migrate_ncit.md
Comment (matentzn) [inline docs/reports/migrate_ncit.md:1]: bit too big to review, noting here that I question the usefulness of this file in general (markdown format I mean)

File: src/ontology/lexmatch/unmapped_ncit_lex.tsv
Comment (matentzn) [inline src/ontology/lexmatch/unmapped_ncit_lex.tsv:1]: This is what I wanted to see!! Awesome!

File: src/ontology/slurp/ncit.tsv
Comment (matentzn) [inline src/ontology/slurp/ncit.tsv:1]: Noting this diff is a bit big for my liking, if you can give it a second glance. Probably nothing.

**Review by matentzn — APPROVED** [review]
I would do one more sanity check (see comments), else this looks great!

File: src/ontology/slurp/ncit.tsv
Comment (twhetzel) [reply src/ontology/slurp/ncit.tsv:1]: Sure, it looks like there are now definitions for the many of the existing rows therefore the large diff. 
Screenshot 2024-08-09 at 12 09 59 PM

Comment (matentzn) [reply src/ontology/slurp/ncit.tsv:1]: Loooks great!

Comment (twhetzel) [reply src/ontology/slurp/ncit.tsv:1]: Thanks, I'll close this and merge https://github.com/monarch-initiative/mondo-ingest/pull/613 into `develop`.

**Pull request closed without merging**